### DISCUSSION TITLE
commit message for fixes on reported issues [ANSWERED - We'll use more informative messages]

### DISCUSSION THREAD

**Discussion by tksoh** [eteran/nedit-ng#217] [General]
I notice a common practice on github that just say "fixes issue #xxx" on the commits, and offer no further details. While it's useful if we are viewing the commit history on github, it will be difficult to figure out what issues those commits are trying to address when we are looking at them offline.

Wouldn't it be more helpful to spell out what the commit really is doing, then add the "fixes issue #xxx" as additional info (assuming github still could pick them up)? For example:

> fix macro issue causing heavy system load (issue #145)

versus

> Should fix issue #145

**Comment by eteran** [comment]
I agree of course. Yes, GitHub does in fact pick up ANY message, so long as it has a hash followed by a number, it will see it as an issue reference. And as a fallback, an issue can always be explicitly linked to a PR (merging the PR will even automagically close the issue it's connected to!).

So yea definitely can develop some better habits regarding git commit message :-).
